tools/README.md

```diff
@@ -86,12 +86,12 @@ python3 /graphstorm/tools/partition_graph.py --dataset ogbn-arxiv \
 
 ## Convert node features from distDGL format to WholeGraph format
 
-Use the `convert_feat_to_wholegraph.py` script with `--dataset-path` pointing to the distDGL folder of partitions. It will convert all the node features and labels to WholeGraph compatible format.
+Use the `convert_feat_to_wholegraph.py` script with `--dataset-path` pointing to the distDGL folder of partitions and `--feat-names` to the features you want to convert (e.g. '`paper/feat,author/feat,institution/feat`'). It will convert all the node features to WholeGraph compatible format.
```

Comment:
What if one node type has multiple feats?
Can you follow: How GraphStorm handle node_feat_name: [https://github.com/awslabs/graphstorm/blob/0a299ee91e3646eb7cdf1bb49cabad921bc32282/python/graphstorm/config/argument.py#L661-L668

You can use ](https://github.com/awslabs/graphstorm/blob/0a299ee91e3646eb7cdf1bb49cabad921bc32282/python/graphstorm/config/argument.py#L1975-L1981)

Reply:
Will resolve in a separate PR (https://github.com/awslabs/graphstorm/issues/457)